Resources/Audio/Animals/attributions.yml

```diff
@@ -0,0 +1,114 @@
+- files: ["monkey_scream.ogg"]
+  license: "CC-BY-NC-SA-4.0"
+  copyright: "Taken or edited from source"
+  source: "https://github.com/goonstation/goonstation"
+
+- files: ["cat_meow.ogg"]
+  license: "CC-BY-4.0"
+  copyright: "Taken or edited from source"
+  source: "https://freesound.org/people/TRNGLE/sounds/368006/"
+
+- files: ["cat_hiss.ogg"]
+  license: "CC0-1.0"
+  copyright: "Taken or edited from source"
+  source: "https://freesound.org/people/Zabuhailo/sounds/146963/"
+
+- files: ["small_dog_bark_happy.ogg"]
+  license: "CC0-1.0"
+  copyright: "Taken or edited from source"
+  source: "https://freesound.org/people/MisterTood/sounds/9032/"
+
+- files: ["duck_quack_happy.ogg"]
+  license: "CC0-1.0"
+  copyright: "Taken or edited from source"
+  source: "https://freesound.org/people/Tabby+Gus./sounds/515408/"
+
+- files: ["chicken_cluck_happy.ogg"]
+  license: "CC0-1.0"
+  copyright: "Taken or edited from source"
+  source: "https://freesound.org/people/Rudmer_Rotteveel/sounds/316920/"
+
+- files: ["ferret_happy.ogg"]
+  license: "CC-BY-4.0"
+  copyright: "Taken or edited from source"
+  source: "https://freesound.org/people/J.Zazvurek/sounds/155115/"
+
+- files: ["mouse_squeak.ogg"]
+  license: "CC0-1.0"
+  copyright: "Taken or edited from source"
+  source: "https://freesound.org/people/Breviceps/sounds/445958/"
+
+- files: ["sloth_squeak.ogg"]
+  license: "CC0-1.0"
+  copyright: "Taken or edited from source"
+  source: "https://freesound.org/people/Higgs01/sounds/428114/"
+
+- files: ["parrot_raught.ogg"]
+  license: "CC-BY-4.0"
+  copyright: "Taken or edited from source"
+  source: "https://freesound.org/people/Coral_Island_Studios/sounds/432588/"
+
+- files: ["fox_squeak.ogg"]
+  license: "CC0-1.0"
+  copyright: "Taken or edited from source"
+  source: "https://freesound.org/people/Breviceps/sounds/468442/"
+
+- files: ["frog_ribbit.ogg"]
+  license: "CC0-1.0"
+  copyright: "Taken or edited from source"
+  source: "https://freesound.org/people/egomassive/sounds/536759/"
+
+- files: ["goose_honk.ogg"]
+  license: "CC0-1.0"
+  copyright: "Taken or edited from source"
+  source: "https://freesound.org/people/SpaceJoe/sounds/510940/"
+
+- files: ["snake_hiss.ogg"]
+  license: "CC-BY-4.0"
+  copyright: "Taken or edited from source"
+  source: "https://freesound.org/people/Garuda1982/sounds/541656/"
+
+- files: ["pig_oink.ogg"]
+  license: "CC0-1.0"
+  copyright: "Taken or edited from source"
+  source: "https://freesound.org/people/qubodup/sounds/442906/"
+
+- files: ["cerberus.ogg"]
+  license: "CC0-1.0"
+  copyright: "Taken or edited from source"
+  source: "https://freesound.org/people/ahill86/sounds/207124/"
+
+- files: ["space_dragon_roar.ogg"]
+  license: "CC-BY-SA-4.0"
+  copyright: "Taken or edited from source"
+  source: "https://github.com/tgstation/tgstation"
+
+- files: ["penguin_squawk.ogg"]
+  license: "CC-BY-4.0"
+  copyright: "Taken or edited from source"
+  source: "https://youtu.be/Anr35RbBL3Y"
```

Comment:
No license stated in the video

Reply:
License was stated in the license.txt but I will try to find the original and confirm the license.

Reply:
Cant find any good replacement. I will add it to the list